tilavarauspalvelu/models/user/queryset.py

```diff
@@ -80,5 +82,17 @@ def anonymize_inactive_users(self) -> None:
             if user.actions.can_anonymize():
                 user.actions.anonymize()
 
+    def should_anonymize_users(self) -> Self:
```

Comment:
Same as the permission deactivation email: it is desirable to notify a user only once, if fresh login is not performed between the condition first triggering for the user and the eventual anonymization taking place.

Reply:
Added field `sent_email_about_anonymization` to user and then checks for this field when gathering email recipients for the email, as well as an update to the after the emails have been sent.